### DISCUSSION TITLE
Welcome to zimbra-build-scripts Discussions!

### DISCUSSION THREAD

**Discussion by ianw1974** [ianw1974/zimbra-build-scripts#42] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jooz** [comment]
Hello, does this script compile the free version, or is it necessary to have a subscription, I currently have version 9 open source from Zetras....

Thank you

**Comment by ianw1974** [comment]
Yes, it compiles OSE/FOSS version, so yes the free one.
